app/assets/stylesheets/layout/_profile.scss

```diff
@@ -75,69 +74,69 @@
   }
 }
 
-// .personal-footer {
-//  background: #fff;
-// }
-// .profile-header {
-//  background: #fdcc55;
-//  min-height: 248px;
-//  border-bottom: 20px $color-white solid;
-// }
-// .profile-wrapper {
-//  // @include make-row();
-//  color: $color-red;
+.personal-footer {
+ background: #fff;
+}
+.profile-header {
+ background: #fdcc55;
+ min-height: 248px;
```

Comment:
Line should be indented 2 spaces, but was indented 1 spaces<br>Properties should be ordered background, border-bottom, min-height